src/main/kotlin/com/example/fhirvalidator/util/ProfileApplier.kt

```diff
@@ -19,7 +19,8 @@ fun getResourcesOfType(resource: IBaseResource, resourceType: String?): List<IBa
 }
 
 fun applyProfile(resources: List<IBaseResource>, profile: IPrimitiveType<String>) {
-    resources.stream()
-        .filter { !it.meta.profile.contains(profile) }
-        .forEach { it.meta.addProfile(profile.value) }
+    resources.stream().forEach {
```

Comment:
Is it worth adding in some tests that it always clears the current profiles and only contains the profile passed in?